books/free-programming-books-hi.md

```diff
@@ -1,10 +1,22 @@
 ### Index
 
 * [C++](#cpp)
+* [Python](#python)
+* [Java](#java)
 
 
 ### <a id="cpp"></a>C++
 
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
+### <a id="python"></a>Python
+
+* [Python in Hindi \| Hindi](https://drive.google.com/file/d/1NX5DHDBmaEacw7bCilQBZ-Lp4A5me6i8/view?usp=sharing) - Kuldeep Chand (PDF)
```

Comment:
Please remove this resource. Drive links are not supported. Please provide only official links

Reply:
I have removed.